Automattic/wp-calypso
# Notifications: Sidebar: Position secondary layout and notifications above tooltip

Fixes #3440 by ensuring that the notes panel and secondary layout show above tooltip.

Also fixes an issue where invalid invite tokens were no longer red. That regression was introduced in d3bd203562068990fe5d3a2c7b8a36fa4ea2c3a5.

To test

- Go to `/people/new/$site` where `$site` is a WP.com site
- Enter in a few invalid users, like `0`, `1`, `2`, etc.
- Ensure there are enough where the notes panel overlays them
- When opening the notes panel, the tooltip should not show through the notes
- Go to mobile width
- Open sidebar
- Ensure that tooltip does not show through
- Open my sites selector
- Ensure tooltip does not show through

cc @gwwar who wrote the z-index docs. I couldn't get the z-index chrome extension to work, so did some guessing and came up with this.

### Screenshots

<img width="428" alt="screen shot 2016-12-07 at 8 09 26 pm" src="https://cloud.githubusercontent.com/assets/1126811/20995053/092fce6c-bcba-11e6-8ebe-adadb0105eb1.png">
<img width="429" alt="screen shot 2016-12-07 at 8 09 32 pm" src="https://cloud.githubusercontent.com/assets/1126811/20995052/092e8188-bcba-11e6-8962-80ac151d0fa9.png">

![screen shot 2016-12-07 at 8 09 38 pm](https://cloud.githubusercontent.com/assets/1126811/20995080/2b3f3a92-bcba-11e6-8180-812031c4d1b3.png)


![screen shot 2016-12-07 at 8 09 09 pm](https://cloud.githubusercontent.com/assets/1126811/20995069/1bdd88f6-bcba-11e6-9da5-ccd6951e176d.png)
